lib/membrane/rtp/ssrc_router.ex

```diff
@@ -179,5 +227,64 @@ defmodule Membrane.RTP.SSRCRouter do
     end
   end
 
+  defp maybe_emit_telemetry_events(actions, state, ctx) do
+    for action <- actions do
+      with {:buffer, {pad, buffer}} <- action do
+        emit_packet_arrival_event(buffer.metadata.rtp.ssrc, buffer.payload, pad, state, ctx)
+      end
+    end
+  end
+
+  defp register_packet_arrival_event(pad, ctx) do
+    Membrane.TelemetryMetrics.register_event_with_telemetry_metadata(
+      telemetry_event_name(),
+      ctx.pads[pad].options.telemetry_metadata
+    )
+  end
+
+  defp emit_packet_arrival_event(ssrc, payload, pad, state, ctx) do
+    Membrane.TelemetryMetrics.execute(
+      telemetry_event_name(),
+      packet_arrival_telemetry_measurements(ssrc, payload, pad, state, ctx),
+      %{ssrc: ssrc, telemetry_metadata: ctx.pads[pad].options.telemetry_metadata}
```

Comment:
Why do we have a `telemetry_metadata` field inside, well, telemetry metadata? :P And why is `ssrc` separate?